samza-api/src/main/java/org/apache/samza/context/Context.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.samza.context;
+
+/**
+ * Container object for all context provided to instantiate an application at runtime.
+ */
+public interface Context {
+  /**
+   * @return framework-provided context for the overall job that is being run
+   */
+  JobContext getJobContext();
```

Comment:
How will the user get hold of `JobContext` & `Context` objects? Will it be added to `ContextManager` API